docs/reference/indices/recovery.asciidoc

```diff
@@ -80,26 +80,27 @@ ID of the shard.
 `type`::
 +
 --
-(String)
-Recovery type.
-Returned values include:
+(String) Recovery source for the shard. Returned values include:
 
-`STORE`::
-The recovery is related to
-a node startup or failure.
-This type of recovery is called a local store recovery.
+`EMPTY_STORE`::
+An empty store. Indicates a new primary shard or the forced allocation of an
+empty primary shard using the <<cluster-reroute,cluster reroute API>>.
 
-`SNAPSHOT`::
-The recovery is related to
-a <<snapshots-restore-snapshot,snapshot restoration>>.
+`EXISTING_STORE`::
+The store of an existing primary shard. Indicates recovery is related
+to node startup, node failure, or the allocation of an existing primary shard.
```

Comment:
Typically not related to node failure, we'd usually promote a replica to primary in that case which doesn't involve recovery.

```suggestion
to node startup or the allocation of an existing primary shard.
```